tour_model_eval/label_processing.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from scipy.cluster.hierarchy import linkage, dendrogram,fcluster
```

Comment:
in terms of code style, I prefer 
`import x.y.z as xyz` instead of `from x.y.z import foo`
that makes it clear where you are importing functions from

Please change.

Reply:
done